### ISSUE TITLE
Flood of xmldom warnings and errors

### ISSUE THREAD

**Issue by asherber** [AnWeber/httpyac#383]
I've used the VS Code extension for a while, but this is my first time with the CLI -- I needed to send a bunch of copies of a certain request at once, to load test a system. The request is one that has worked fine for me in VS Code for months, and when I use the CLI to send it once it also works fine. But as soon as I add `--repeat x` to the CLI, where `x >=2`, my console fills with a flood of messages that look like this:

```
[xmldom warning]        attribute space is required"type"!!
@#[line:120,col:2219]
[xmldom warning]        attribute value must after "="
@#[line:120,col:2219]
[xmldom error]  element parse error: Error: invalid attribute:Drug\
```

I even added `--silent` and `-o none` to the CLI to try to suppress this, but that didn't have any effect. The volume of messages seems proportional to the value of `x`, although even for `--repeat 2` the messages seem far longer than the request itself.

The requests are correctly sent, despite the warnings and errors.

**Comment by asherber** [comment]
Based on the messages, they actually refer to the request, not the response. I can try to sanitize the request and post it here. 

I was also wondering why these messages only occur when I'm sending the request more than once.

**Comment by AnWeber** [comment]
I do not need an example anymore. I found a testcase in xmldom with this error messsage and could reproduce the issue. It will be fixed with the next release.

**Comment by AnWeber** [comment]
Issue is fixed with new update of httpyac (v5.10.2)
